src/main/java/org/sagebionetworks/bridge/models/surveys/Survey.java

```diff
@@ -16,7 +18,11 @@
     static Survey create() {
         return new DynamoSurvey();
     }
-
+    
+    static Survey createHibernate() {
+        return new HibernateSurvey();
+    }
```

Comment:
Doesn't need this. If a class cares specifically about the Hibernate implementation (eg the HibernateSurveyDao), it can create a HibernateSurvey directly.